content/en/platform/corda/5.0-beta/deploying/deployment-tutorials/manual.md

```diff
@@ -56,6 +56,17 @@ For example:
 corda-cli.sh topic -b kafka-1.example.com -k config.properties create -r 3 -p 10 connect
 ```
 
+If you are authenticating Kafka users, the CLI can also create Access Control List (ACL) entries as appropriate for each Corda worker.
+Specify a set of name-value pairs giving the Kafka username that will be used for each Corda worker:
+
+```shell
```

Comment:
In newer content I've been giving the Windows version (`corda-cli.cmd topic`) displayed in tabs. Do you think that's necessary? If so, I will update all instances on this page..

Reply:
It probably is. If nothing else, I've split some of the longer commands over multiple lines using `\` which should also be swapped for the Windows equivalent.